src/AppShed/Extensions/SpreadsheetBundle/Service/GeoService.php

```diff
@@ -11,33 +12,43 @@
  */
 class GeoService {
 
-    public function getGeo($address)
-    {
-        $guzzle = new Client();
-
-        $address = urlencode($address);
+    private $geoClient;
 
-        $url = "http://maps.google.com/maps/api/geocode/json?sensor=false&address={$address}";
+    public function __construct(Client $geoClient)
+    {
+        $this->geoClient = $geoClient;
+    }
 
-        $res = $guzzle->get($url);
+    public function getGeo($address)
+    {
 
-        if($res->getStatusCode()==200){
+        try {
+            $geoResponse = $this->geoClient->get(
+                '',
+                [
+                    'query' => [
+                        'address' => $address
+                    ]
+                ]
+            );
+
+            if ($geoResponse->getStatusCode() != 200) {
+                return false;
+            }
 
-            $resp = $res->json();
+            $resp = $geoResponse->json();
 
-            if ($resp['status']=='OK') {
+            if ($resp['status'] != 'OK' || !count($resp['results'])) {
+                return false;
+            }
 
-                $lati = @$resp['results'][0]['geometry']['location']['lat'];
-                $longi = @$resp['results'][0]['geometry']['location']['lng'];
+            return $resp['results'][0]['geometry']['location'];
 
-                if($lati && $longi){
-                    return [$lati, $longi];
-                }
+        } catch(RequestException $e ) {
 
-            }
+            return false;
```

Comment:
I have just thought as well, its probably good idea to inject `@logger` into this service and log this error before returning.